### ISSUE TITLE
ench soso "golds" and "es"

### ISSUE THREAD

**Issue by PRO100Zloy** [kolton/d2bot-with-kolbot#2482]
Hi ,

what do you have to set for commands that the enchsoso gives
**"golds"** and get **"es"**, **"wps","chant","cows"** all ready working but willl create gold+es

**Comment by gtoilet** [comment]
https://pastebin.com/TuQ9YD5e


see inside

**Comment by PRO100Zloy** [comment]
t

> https://pastebin.com/TuQ9YD5e
> 
> see inside

thanks gtoilet
coming this error 19:59:48 (ench)  Error in Enchant (enchant.js #1177) Config.Enchant.Triggers[4] is undefined (Area: 1, Ping:62, Game: Ench-13)

**Comment by gtoilet** [comment]
your usein the newest version correct

**Comment by PRO100Zloy** [comment]
> your usein the newest version correct
> 
> just stuck script in and got no errors

yes got a newest version , but i will testing try to download again

**Comment by PRO100Zloy** [comment]
> your usein the newest version correct
> 
> just stuck script in and got no errors

new newest version download
 create new ench soso
opening game with d2lead
other player come in game
write wps
Error in Enchant (enchant.js #981) Config.Enchant.Triggers[0] is undefined (Area: 1, Ping:47, Game: Ench-1)

**Comment by gtoilet** [comment]
something has to be wrong on your end for sure ... did cows/chant/wps


[img]https://i.imgur.com/LtXWs12.jpg[/img]
